app/controllers/webhooks_controller.rb

```diff
@@ -139,19 +141,14 @@ def handle_payment_intent_succeeded(
         merchant_name = Seller.find_by(seller_id: seller_id).name
         case item_json['item_type']
         when 'donation'
+          is_donation ||= true
```

Comment:
Maybe this is a ruby quirk i’m not familiar with, but why are we setting this to nil initially instead of false, then `||=`’ing it to true instead of just setting it?

Reply:
Because if I initially set it to false, then set it to true in the loop without using the conditional assignment operator, is_donation will keep getting re-assigned to true after every loop, which is like negligibly more time expensive, but still haha. I COULD use an if-statement but would look kind of ugly haha. Open to discussion though!

<img width="1680" alt="Screen Shot 2020-06-17 at 3 58 15 PM" src="https://user-images.githubusercontent.com/45523555/84959061-79f5ae00-b0b3-11ea-8618-fc7712cf25db.png">

Reply:
Interesting, makes sense that `||=` is faster than a raw assignment. Where’s the advantage come from the initial `nil` vs `false` though?

Reply:
Ahhh I had assumed that if a = false then a ||= true would still give a = false but turns out that isn't the case haha. I'll change that. Thanks man!